sklearn/__init__.py

```diff
@@ -21,8 +21,7 @@
 from ._config import get_config, set_config, config_context
 
 logger = logging.getLogger(__name__)
-logger.addHandler(logging.StreamHandler())
-logger.setLevel(logging.INFO)
+logger.addHandler(logging.NullHandler())
```

Comment:
Why not remove the `logger.addHandler` line altogether and leave it to default handler?

Reply:
It has to do with the behavior of `WARNING`:

> If the using application does not use logging, and library code makes logging calls, then (as described in the previous section) events of severity WARNING and greater will be printed to sys.stderr. This is regarded as the best default behaviour.

Anyways I am okay with removing the handler as well.